neqo-transport/src/qlog.rs

```diff
@@ -127,6 +128,44 @@ pub fn connection_state_updated(qlog: &mut NeqoQlog, new: &State) {
     })
 }
 
+pub fn client_version_information_initiated(qlog: &mut NeqoQlog, version_config: &VersionConfig) {
+    qlog.add_event_data(|| {
+        let ev_data = EventData::VersionInformation(VersionInformation {
+            client_versions: Some(
+                version_config
+                    .all()
+                    .iter()
+                    .map(|v| format!("{:02x}", v.wire_version()))
+                    .collect(),
+            ),
+            server_versions: None,
+            chosen_version: Some(format!("{:02x}", version_config.initial().wire_version())),
+        });
+        Some(ev_data)
+    });
+}
+
+pub fn client_version_information_negotiated(
+    qlog: &mut NeqoQlog,
+    client: &[Version],
+    server: &[WireVersion],
+    chosen: Version,
+) {
+    qlog.add_event_data(|| {
+        let ev_data = EventData::VersionInformation(VersionInformation {
```

Comment:
You can inline `ev_data` (it's less code overall).

Reply:
Sounds good. Done in https://github.com/mozilla/neqo/pull/1505/commits/a5f7681671085ab0fa3f65bf8eb94927e6c44f4f.